### PR TITLE
fix some bugs appeared in firsts executions

### PR DESCRIPTION
**Pull request by Olcina**
in readme: Change variable name length to num_frames 
in mediafundation: added re module to the import statements

### CODE REVIEW

**Comment by bastibe** [comment]
Thank you very much for your pull request!

File: README.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ default_speaker.play(data/numpy.max(data), samplerate=44100)
 # alternatively, get a `recorder` and `player` object and play or record continuously:
 with default_mic.recorder(samplerate=44100) as mic, default_speaker.player(samplerate=44100) as sp:
     for _ in range(100):
-        data = mic.record(length=1024)
+        data = mic.record(num_frames=1024)
```
Comment (bastibe) [inline README.md]: should be `numframes`

**Comment by bastibe** [comment]
I'll gladly merge if you change the above line.

**Comment by Olcina** [comment]
Hello,

Yes with those changes works for me.  I found your blog when looking for a python way of connect my guitar to the computer and add some gain distortion to the line sound.

I've made the changes to change the variable 'num_frames' to 'numframes'

Thanks to you for your work!

**Comment by bastibe** [comment]
That's embarrassing ;-)

Well, thank you very much for fixing those bugs!

**Pull request merged**